tests/utils/inputparams/testparams.go

```diff
@@ -78,6 +78,14 @@ func GetUniqueVolumeName(volName string) string {
 	return normalizeVolumeName(volName + "_volume_" + GetRandomNumber())
 }
 
+// GetSharedVolumeName prepares unique volume name that is DNS compliant
+// Shared volumes launch docker service. Name of such a service must be
+// DNS compliant, hence volname should also be DNS compliant.
```

Comment:
Here is the error that the shared plugin throws while creating shared volumes:

2017-08-16 13:50:28.530790104 -0700 PDT [WARNING] Failed to create file server for volume AdvancedSharedTestSuite.TestSharedVolumeLifecycle_volume_495314. Reason: Error response from daemon: {"message":"rpc error: code = 3 desc = name must be valid as a DNS name component"}